start_nevermined.sh

```diff
@@ -590,21 +591,13 @@ while :; do
             printf $COLOR_Y'Starting with local Polygon node...\n\n'$COLOR_RESET
             ;;
         # spins up geth dev mode
-        --geth)
+        --geth | --local-node | --spree-embedded-contracts)
```

Comment:
I would just remove `--spree-embedded-contracts` altogether because it uses a different network name and chainId and I'm afraid this will just cause problems